pyproject.toml

```diff
@@ -160,3 +160,6 @@ max-args=10
 [tool.pytest.ini_options]
 minversion = "6.0"
 addopts = "--verbose --durations=0 --durations-min=0.2 --doctest-modules --mpl --mpl-results-path=results"
+markers = [
+    "benchmark: mark a test with custom benchmark settings.",
```

Comment:
For reference: text is copied from https://github.com/ionelmc/pytest-benchmark/blob/cec69f5fcf793321fb1ec5c961bd2c6a3953a3c0/src/pytest_benchmark/plugin.py#L482